### PR TITLE
Specify supported R-version in documentation

### PR DESCRIPTION
**Pull request by sidohaakma**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

**Comment by dennishendriksen** [comment]
Does https://github.com/molgenis/docker meet this requirement?

**Comment by sidohaakma** [comment]
No, opencpu/base (the latest runs on R 3.4.1). Maybe we should specify a version of the opencpu docker?

**Comment by dennishendriksen** [comment]
How about requiring R 3.4.x instead of 3.2 for MOLGENIS?

**Comment by sidohaakma** [comment]
That is a good idea. We need to perform a 2 steps then:
- require the bit (this package contains the method chunk in R 3.4.1) in the molgenis.R script
- check if the package is available on opencpu (golden vm and docker)

**Comment by dennishendriksen** [comment]
FYI from http://www.molgenis.org/wiki/GccSopCreateMolgenisVM:
```
# install R libraries
Rscript -e 'install.packages(c("bitops", "RCurl", "rjson", "gdata", "corrplot", "qqman", "ggplot2", "ff", "httr"), repos="http://cran-mirror.cs.uu.nl/")'
```

**Comment by sidohaakma** [comment]
Now concluded we need to add to the documentation that only the molgenis.R client is comptabli with R-versions up to 3.2.1.

File: docs/developer_documentation/ref-R.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ source("http://molgenis.mydomain.example/molgenis.R")
 ``` 
 at the top of your script and you can connect to a MOLGENIS server. Typically the first thing you do is login and the last thing is logout.
 
-NOTE: For https connections use 
+**NOTE:** For https connections use 
 ```
 eval(expr = parse(text = getURL("https://molgenis.mydomain.example/molgenis.R?molgenis-token=${molgenisToken}")))
 ```  
 
+**NOTE:** We support R-versions earlier or equal to 3.2.x
```
Comment (fdlk) [inline docs/developer_documentation/ref-R.md]: Could you make it a little bit clearer that this specifically refers to the R api client?

**Comment by fdlk** [comment]
@sidohaakma what's the status of this one?

**Comment by sidohaakma** [comment]
@dennishendriksen I have specified the R-api supported R version in the documentation.

**Pull request merged**